lib/graphql/schema/field.rb

```diff
@@ -355,7 +357,7 @@ def initialize(type: nil, name: nil, owner: nil, null: nil, description: :not_gi
       # @return [Boolean, nil]
       # @see GraphQL::Subscriptions::BroadcastAnalyzer
       def broadcastable?
-        if defined?(@broadcastable)
+        if @broadcastable != NOT_CONFIGURED
```

Comment:
Probably not worth it, but when using this kind of "unset" tokens, I prefer to inverse the check so that if I hit a weird user defined object with a custom `==` it won't be invoked.

You can also use `equal?` to only check identity:

```suggestion
        if !NOT_CONFIGURED.equal?(@broadcastable)
```

But I agreed that it doesn't read as well.

Reply:
:+1: Thanks for the suggestion, I'll update it!